web/mocks/api/v2/finex/public/markets/GET.mock

```diff
@@ -22,7 +22,19 @@ Access-Control-Allow-Credentials: true
         "digits":5,
         "type":"significant_digits"
       }
-    ]
+    ],
+    "features": {
+      "order_types": [
+        "fok",
+        "limit",
+        "market",
+        "fok",
```

Comment:
could you please remove duplicating 'fok'

Reply:
Removed duplicate fok order type